PublicMapping/districtbuilder-classic
# Fix gitkeep file for data directory

## Overview

A `.gitkeep` file was placed in the root directory, but it actually needed to go in the `data` directory in order to ensure that the bare `data` directory is part of the repo. 

### Notes

`.gitkeep` is just a convention to create an empty directory and isn't an official part of `git`.

## Testing Instructions

 * Remove and `data` directory you may have already created.
 * Pull the code and ensure a `data` directory is created.
 * Add a zip file to the `data` directory and ensure `git status` does not see any local changes.
